src/batch-client/index.ts

```diff
@@ -841,7 +890,37 @@ export class ZimbraBatchClient {
 		this.jsonRequest({
 			name: 'GetIdentities',
 			namespace: Namespace.Account
-		}).then(res => mapValuesDeep(res, coerceStringToBoolean));
+		}).then(({ identity, ...restResult }: any) => {
+			const updatedIdentity: any = identity.map(
+				({
+					_attrs: {
+						zimbraPrefWhenInFolderIds,
+						zimbraPrefWhenSentToAddresses,
+						...restAttrs
+					},
+					...restIdentity
+				}: any) => ({
+					...restIdentity,
+					_attrs: {
+						...restAttrs,
+						zimbraPrefWhenInFolderIds: []
```

Comment:
convertStringAndArrayValues

Reply:
For `getIdentities`, On client side, we need arrays only. So not converting using this method. Added code comment.